### DISCUSSION TITLE
Progression (Add set) function

### DISCUSSION THREAD

**Discussion by AdEmergency625** [astashov/liftosaur#140] [Ideas]
Many top coaches consider adding sets across a cycle to be a good way to drive hypertrophy progression. It would be great to have a simple function to add a set each week or a certain number of workouts.

**Comment by yarox** [comment]
it'll be nice if the app could add or take away sets from an exercise given how much you have recovered between sessions. the key word here is "recovered"; i don't think that there is a way to track this currently.

**Comment by astashov** [comment]
You can do it now via `numberOfSets += 1`
